generator/beat/{beat}/beater/{beat}.go.tmpl

```diff
@@ -18,15 +18,15 @@ type {Beat} struct {
 }
 
 // Creates beater
-func New(b *beat.Beat, cfg *common.Config) (beat.Beater, error) {
-	config := config.DefaultConfig
-	if err := cfg.Unpack(&config); err != nil {
+func New(b *beat.Beat, c *common.Config) (beat.Beater, error) {
```

Comment:
What is the reason for the renaming here?

Reply:
this stops shadowing the `config` import

Reply:
I see. Seems like we have that also in our beats.

The reason the renaming is a bit unfortunate is because in most of our code base we use `cfg` to represent go-ucfg / common.Config type. WDYT think about using `c := config.DefaultConfig` instead and keeping `cfg` as it is?

Reply:
Sounds sane, I'll make that update